categories.py

```diff
@@ -78,6 +78,30 @@
   "francescobrivio":  ["alca"],
   "malbouis":         ["alca"],
   CMSBUILD_USER:      ["tests" ],
+  # dpgs
+  "tsusa":            ["trk_dpg"],
```

Comment:
To me using hyphen as the separator along
```suggestion
  "tsusa":            ["trk-dpg"],
```
would work better with the GitHub team names (`@cms-sw/trk-dpg-l2`) and for the labels in case a PR/issue is assigned (`trk-dpg-pending`, `trk-dpg-approved`, `trk-dpg-rejected`).